py/picca/delta_extraction/data.py

```diff
@@ -400,34 +442,31 @@ def find_nside(self):
         for forest, healpix in zip(self.forests, healpixs):
             forest.healpix = healpix
 
-    def save_deltas(self):
+    def save_deltas(self, num_processors=1):
```

Comment:
can't we actually use `self.num_processors` inside the function? We'd just need to add a `self.num_processors=1` to the `__init__` here (as DESI data has it and SDSS does not)

Reply:
I agree, I'd say we move the definition of `self.num_processors` from `DesiData` to `Data`. This way, we will be implementing the parallelization of the saving also for `SdssData`. This change will not otherwise affect `SdssData` as this variable would not be used elsewhere

Reply:
We will need to update the data model, though

Reply:
I tried it, but it got very complicated. This option is entangled in configuration settings. Maybe just merge this, and we figure out how to move design later?

Reply:
I fixed it in PR #921. That PR is merging to this branch so it should be resolved first